app/helpers/denounce_helper.rb

```diff
@@ -0,0 +1,15 @@
+module DenounceHelper
+
+  def options_for_state_select
+    State.names
+  end
+
+  def options_for_municipality_select(address)
+    address ? Municipality.for(address.state).pluck(:name) : []
+  end
+
+  def options_for_settlement_select(address)
+    address ? Settlement.for(address.municipality).pluck(:name) : []
+  end
+
```

Comment:
Favor de remover las líneas blancas extra al inicio y al final del módulo. Gemas como `rubocop` se suelen quejar de esto.

Reply:
Will do, thanks for the observation.